### PR TITLE
WRN-16301: Drawing board setup

### PR DESCRIPTION
**Pull request by adrian-cocoara-lgp**
Enact-DCO-1.0-Signed-off-by: Adrian Cocoara adrian.cocoara@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Added drawing board setup

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-16301

### CODE REVIEW

File: packages/ui/DrawingCanvas/Drawing.js [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+	handlers: {
+		draw: (event, {points}) => {
+			const {
+				beginPointRef,
+				contextRef,
+				ev,
+				isDrawing,
+				isErasing,
+				setIsDrawing,
+				offset
```
Comment (daniel-stoian-lgp) [inline packages/ui/DrawingCanvas/Drawing.js:70]: sort

File: packages/ui/DrawingCanvas/Drawing.js [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+		startDrawing,
+		finisDrawing,
+		draw,
+		drawingRef,
+		disabled,
+		isErasing,
+		brushColor,
+		brushSize,
+		canvasColor,
+		...rest
```
Comment (daniel-stoian-lgp) [inline packages/ui/DrawingCanvas/Drawing.js:143]: sort props

File: packages/ui/DrawingCanvas/Drawing.js [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+		}, []); // eslint-disable-line react-hooks/exhaustive-deps
+
+		useEffect(() => {
+			const canvas = canvasRef.current;
+
+			const context = canvas.getContext('2d');
+			context.lineWidth = brushSize;
+			context.strokeStyle = brushColor;
+		}, [brushSize, brushColor]);
+
```
Comment (vladut-clapou-lgp) [inline packages/ui/DrawingCanvas/Drawing.js:177]: Can you order the dependencies alphabetically please?

**Review by vladut-clapou-lgp — APPROVED** [review]
LGTM

**Pull request merged**